packages/nextjs-kit/src/components/header.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Link from 'next/link';
+import { Headers } from '../lib/interfaces';
+
+interface HeaderProps {
+  headers: Headers[];
+}
+
+/**
+ * This is a Header component.
+ *
+ * @param {Props} headers holds the title and href of each route to be displayed
+ * @returns React.JSX.Element
```

Comment:
```suggestion
 * @returns {React.JSX.Element}
```
Wrapping the type in `{}` should help our TSDoc definitions